packages/core/config/webpack.common.js

```diff
@@ -226,7 +226,10 @@ module.exports = (webpackEnv) => {
           // @TODO: Add redirect to fatalErrorPage
           // lastResortScript: "window.location.href='/500.html';"
         }),
-      isEnvProduction && new WebpackManifestPlugin(),
+      isEnvProduction &&
+        new WebpackManifestPlugin({
+          fileName: 'rw-build-manifest.json',
```

Comment:
how about just 'build-manifest.json`. the `rw` feels redundant, yes?

Reply:
You got it!